app/backend/server/api/v1/dates_router.py

```diff
@@ -26,6 +23,35 @@
                                               end_date=end,
                                               users_id=users_id)
 
+@router.post("/", response_model=schemas.RangeDatesResponse)
+async def post_dates_status(
+    body: schemas.PostDateDataBody,
+    db: Session = Depends(get_db),
+    current_user: schemas.User = Security(get_current_user,
+                                          scopes=["personal"])
+):
+    return crud.set_new_date_data(db=db, body=body)
+
+@router.delete("/")
+async def delete_dates_status(
+    start: date, 
+    end: date = None, 
+    users_id: List[int] = Query([]), 
+    db: Session = Depends(get_db)
+):
+    crud.delete_users_dates_data(db=db,
+                            start_date=start, 
+                            end_date=end, 
+                            users_id=users_id)
+
+@router.put("/", response_model=schemas.RangeDatesResponse)
+async def put_dates_status(
+    body: schemas.PutDateDataBody,
+    db: Session = Depends(get_db),
+    current_user: schemas.User = Security(get_current_user,
+                                          scopes=["personal"])
+):
+    return crud.put_data_in_user(db=db, body=body)
```

Comment:
expand body here too

Reply:
get rid of the PutDateDataBody here and everywhere

Reply:
done in the expand part